.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,41 @@
+---
+name: Bug report
```

Comment:
Ask users to include a timestamp/timezone of the incident.  It can help if there are logs that can be searched.

Reply:
timestamptz field added for incident reporting